main/src/cgeo/geocaching/filters/core/UserDisplayableStringUtils.java

```diff
@@ -0,0 +1,27 @@
+package cgeo.geocaching.filters.core;
+
+final class UserDisplayableStringUtils {
+
+    private UserDisplayableStringUtils() {
+        // Do not instantiate
+    }
+
+    static String getUserDisplayableConfig(final boolean minValueSet, final boolean maxValueSet, final String minValue, final String maxValue) {
```

Comment:
I believe you could leave out the parameters "minValueSet" and "maxValueSet" and instead make the parameters "minValue" and "maxValue" nullable. If "minValue" is null, then minValue is not set, same for maxValue. I believe this would make the code a bit easier, also on the caller side

Reply:
good point